instaclustr/resource_cluster.go

```diff
@@ -26,6 +26,11 @@ func resourceCluster() *schema.Resource {
 		Read:   resourceClusterRead,
 		Update: resourceClusterUpdate,
 		Delete: resourceClusterDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceClusterStateImport,
+		},
+
```

Comment:
The API response doesn't fully support all of the required fields/structure needed for importing clusters. Namely missing support for Kafka, Redis, etc.